addons/modules/functions/fnc_moduleUnpackStaticWeapon.sqf

```diff
@@ -0,0 +1,73 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to unpack and assemble a static weapon from supported backpacks.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleUnpackStaticWeapon
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _gunner = attachedTo _logic;
+deleteVehicle _logic;
+
+if (isNull _gunner) exitWith {
+    [LSTRING(NoUnitSelected)] call EFUNC(common,showMessage);
+};
+
+if !(_gunner isKindOf "CAManBase") exitWith {
+    [LSTRING(OnlyInfantry)] call EFUNC(common,showMessage);
+};
+
+if !(alive _gunner) exitWith {
+    [LSTRING(OnlyAlive)] call EFUNC(common,showMessage);
+};
+
+if (isPlayer _gunner) exitWith {
+    ["str_a3_cfgvehicles_moduleremotecontrol_f_errorPlayer"] call EFUNC(common,showMessage);
+};
+
+private _fnc_getCompatibleBases = {
+    params ["_backpack"];
+    private _cfgBase = configFile >> "CfgVehicles" >> _backpack >> "assembleInfo" >> "base";
+    if (isText _cfgBase) exitWith { toLower (getText _cfgBase) };
+    (getArray _cfgBase) apply {toLower _x}
+};
+
+private _backpack = backpack _gunner;
+
+if (_backpack isEqualTo "") exitWith {
+    [LSTRING(ModuleUnpackStaticWeapon_Unit)] call EFUNC(common,showMessage);
+};
+
+private _compatibleBases = [_backpack] call _fnc_getCompatibleBases;
+if (_compatibleBases isEqualType "") then {_compatibleBases = [_compatibleBases];};
```

Comment:
I think `_fnc_getCompatibleBases` should take care of this.

Reply:
https://github.com/zen-mod/ZEN/pull/378/commits/d8d7591a49eaf95fd249f6e9d43add21b3dc591b